3.5/hot-backup-restore.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+description: Hot backup and restore and associated operation may be performed via the tool arangobackup.
+title: ArangoDB Hot Backup and Restore Overview
+---
+Hot Backup and Restore
+======================
+
+Hot backup and restore and associated operations may be performed tool
+[_arangobackup_](programs-arangobackup.html).
+
+Many operations cannot afford downtimes and thus require administrators and
+operators to create consistent freezes of the data during normal operation.
+Such use cases imply that near instantaneous backups must be obtained in sync
+across say a cluster's deployment. For this purpose the hot backup mechanism
+was created.
+
+In the following the ArangoDB hot backup feature is explained in some detail.
+It is important to understand the technology as best as possible in order to
+conclude which operations it might or might not be suited for.
+
+{% hint 'warning' %}
+Please review the requirements and limitations of hot backups, specifically
+regarding storage engine, deployment, scope and storage space.
+{% endhint %}
```

Comment:
Redundant warning. it is also in 3.5/hot-backup-restore-introduction.md with a link to the limitations page. Can we remove this one?